sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerClientJavaDocCodeSnippets.java

```diff
@@ -947,7 +947,7 @@ public void beginRecognizeIdentityDocumentsFromUrl() {
                 FormField countryFormField = recognizedFields.get("Country");
```

Comment:
same here. Anywhere where u had `Country` triple check that it is now `CountryRegion`

Reply:
applies to all samples and tests